### PR TITLE
add 1.37

### PR DESCRIPTION
**Pull request by gesinn-it-gea**
Make MW 1.37 available as docker image

### CODE REVIEW

File: 1.37/apache/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,116 @@
+FROM php:7.4-apache
```
Comment (davidbarratt) [inline 1.37/apache/Dockerfile:1]: should we bump this to 8.0 ?

Comment (davidbarratt) [reply 1.37/apache/Dockerfile:1]: oh nevermind https://phabricator.wikimedia.org/T248925

**Comment by malberts** [comment]
I believe 1.37 should be added to the GitHub action as well:
https://github.com/wikimedia/mediawiki-docker/blob/master/.github/workflows/build.yml#L17

**Comment by gesinn-it-gea** [comment]
@davidbarratt can you merge this PR? It would help the SMW community since we use mediawiki-docker in CI and want to run CI tests as early as possible against recent MW.

**Comment by davidbarratt** [comment]
@gesinn-it-gea oops, hit the merge a little too quickly, can you add it to the GitHub Action?
https://github.com/wikimedia/mediawiki-docker/blob/master/.github/workflows/build.yml#L17

**Comment by gesinn-it-gea** [comment]
> I believe 1.37 should be added to the GitHub action as well: https://github.com/wikimedia/mediawiki-docker/blob/master/.github/workflows/build.yml#L17

uh, completely missed your comment @malberts . Thx for fixing it already!

**Pull request merged**